content/scripts/build.js

```diff
@@ -59,6 +59,16 @@ function isTTY() {
   return !!process.stdout.columns;
 }
 
+// Turn a Map instance into a object.
+// This is something you might need to do when serializing a Map
+// with JSON.stringify().
+function mapToObject(map) {
```

Comment:
I'm thinking you don't need this function based on my `JSON.stringify` comment, but if you do, I wonder how this compares to just using `{...map}`, which works too but doesn't nullify the prototype.

Reply:
Never mind! I'm wrong!